opal/mca/btl/usnic/btl_usnic_module.c

```diff
@@ -1436,6 +1438,21 @@ static int create_ep(opal_btl_usnic_module_t* module,
         channel->info->caps &= ~(1ULL << 63);
     }
 
+    /* all of the OMPI code assumes IPv4, but some versions of libfabric will
+     * return FI_SOCKADDR instead of FI_SOCKADDR_IN, so we need to do a little
+     * bit of sanity checking */
+    assert(FI_SOCKADDR_IN == channel->info->addr_format ||
+           FI_SOCKADDR == channel->info->addr_format);
+    if (FI_SOCKADDR == channel->info->addr_format) {
```

Comment:
Per the assert right above it, does this "if" need to check for FI_SOCKADDR or FI_SOCKADDR_IN?